test_agent/cpp/conanfile.py

```diff
@@ -0,0 +1,67 @@
+from conan import ConanFile
+from conan.tools.cmake import CMake, CMakeToolchain, cmake_layout
+from conan.tools.files import apply_conandata_patches, copy, export_conandata_patches, get, replace_in_file, rm, rmdir
+import os
+
+class TestAgentRecipe(ConanFile):
+    name = "test_agent_cpp"
+    package_type = "executable"
+    license = "Apache-2.0 license"
+    homepage = "https://github.com/eclipse-uprotocol"
+    url = "https://github.com/conan-io/conan-center-index"
+    description = "C++ Test Agent"
+    #topics = ("ulink client", "transport")
+    # Binary configuration
+    settings = "os", "compiler", "build_type", "arch"
+    options = {"shared": [True, False], "fPIC": [True, False]}
```

Comment:
There seems to be double declaration of option. We should be able to safely drop this one.

Reply:
done